src/gcp_scanner/crawl.py

```diff
@@ -218,11 +212,8 @@ def get_static_ips(project_name: str,
     request = service.addresses().aggregatedList(project=project_name)
     while request is not None:
       response = request.execute()
-      for name, addresses_scoped_list in response["items"].items():
-        if addresses_scoped_list.get("addresses", None) is None:
-          continue
-        ips_list.append({name: addresses_scoped_list})
-
+      ips_list = [{name: addresses_scoped_list} for name, addresses_scoped_list in response["items"].items() if addresses_scoped_list.get("addresses", None) is not None]
```

Comment:
Split to multiple lines for readability.

Reply:
Hi @ZetaTwo , I have resolved this issue in the latest PR.